schemas/questionnaire_v1.json

```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
```

Comment:
The `survey` property should be removed.

Reply:
Removed, thanks!